### ISSUE TITLE
Multiscale Deep Equilibrium Models 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#90]
# TL;DR
Segmentation, ImageNetで高精度を達成できる They propose an MDEQ that extends DEQ, which can be trained at practically infinite depths, to image-based tasks. By using equilibrium states at multiple resolutions, one model can support multiple tasks. The actual depth is only one ResBlock. And high Accuracy can be achievedthed on segmentation, ImageNet.

![image](https://user-images.githubusercontent.com/24524018/92561356-0bd61700-f2af-11ea-833f-d39228224296.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2006.08656

# Submission Dates(yyyy/mm/dd)

# Authors and institutions

# Methods

# Results

# Comments
